ApollosProject/apollos-apps
# fix: android header issues

## ✏️ Description


This PR attempts to address some ongoing issues we are having with Android header avoidance. Right now we have content, featureFeeds, and searchConnected that is being hidden under the header. This adds back the 'avoidHeader' prop to solve the featureFeed issue, insetHeight to solve the content issue, and safeArea 'top' to solve the search issue. Because, these components are all formed slightly differently and in different context, it was necessary to use three different solves instead of just one 😢 . Open to suggestions if the reviewer sees a way to just use two (or even 1!) solve instead of 3.

## 🔬 To Test
1. Pull down this PR
2. Start up the Android sim and check: User Settings, Personal Details, Prayer, Search, Content Items, and Tab Feature Feeds.  

## 📸 Screenshots

|Location|Before|After|
|---|---|---|
|Home|![Screenshot 2022-11-30 at 7 49 33 PM](https://user-images.githubusercontent.com/72768221/204946748-eb4d12fe-64c4-458d-ac0c-63b674a5ed5c.png)|![Screenshot 2022-11-30 at 7 34 12 PM](https://user-images.githubusercontent.com/72768221/204945946-d4d936a9-3941-4169-baca-9e29c7121087.png)|
|Search|![Screenshot 2022-11-30 at 7 49 51 PM](https://user-images.githubusercontent.com/72768221/204946824-f456805f-a61c-42a5-9e4f-051bce92c096.png)|![Screenshot 2022-11-30 at 7 34 36 PM](https://user-images.githubusercontent.com/72768221/204946098-bf60f614-301d-48b8-9d18-41e75bc08f54.png)|
|Content|![Screenshot 2022-11-30 at 7 50 05 PM](https://user-images.githubusercontent.com/72768221/204946833-750d2aef-cba4-4bcf-8aef-eb8831c84c33.png)|![Screenshot 2022-11-30 at 7 35 09 PM](https://user-images.githubusercontent.com/72768221/204946170-735934eb-c1b8-44d1-acc4-1845cb9f1c4f.png)|
|User Settings|![Screenshot 2022-11-30 at 7 50 17 PM](https://user-images.githubusercontent.com/72768221/204946852-c1026cfd-0d39-4b9b-b3b8-e4034fdb4cb0.png)|![Screenshot 2022-11-30 at 7 35 18 PM](https://user-images.githubusercontent.com/72768221/204946216-c4e74957-ad7f-4c94-ae9e-d0131c98acb5.png)|
|Personal Details|![Screenshot 2022-11-30 at 7 50 27 PM](https://user-images.githubusercontent.com/72768221/204946867-98fa8774-e205-48e3-a430-42b94101307f.png)|![Screenshot 2022-11-30 at 7 35 27 PM](https://user-images.githubusercontent.com/72768221/204946264-4eabe4d8-b5ce-48c5-8897-efe67d1eb89e.png)|

## ⚠️ To-do Before Merge
None


## 📝 Checklist
